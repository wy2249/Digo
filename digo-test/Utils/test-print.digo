func digo_main() void {
  println("%d", 42)
  println("%f", 42.012)
  println("%s", "hello")
}
