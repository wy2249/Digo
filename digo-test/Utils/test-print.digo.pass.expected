42
42.012000
hello
